lib/systems/src/PedalsSystem.cpp

```diff
@@ -0,0 +1,185 @@
+#include <math.h>
+#include <tuple>
+#include "PedalsSystem.h"
+
+// VCF interface/system data - has all pedals data - global. Analog readings of 4 pedals sensors - all data that is in pedal system data
```

Comment:
Please rewrite this comment to comply with the other comment headers. Either doxygen or javadoc style.